final_writeup.md

```diff
@@ -0,0 +1,37 @@
+
+Look at these videos by clicking on the links below.  
+[Functional Example](https://goo.gl/photos/sJZFpJsXKpvtFuUNA)  
+[Disfunctional Example](https://goo.gl/photos/BtXQyyNE8zBDnFfC9)  
+
+The first video gives step-by-step directions on how to use the program and also shows what happens when there are no images saved from the webcam. If the user does not capture any snapshots from the webcam, then the music will not play, stop, or shuffle because of the conditions we included. This is shown in the second video. Files with the names we specified must exist (the statement is true) in order for the user to listen to any music.
+
+**Implementation**
+
+At a system architecture level, there are multiple steps the data goes through before the music stops. First, the user poses in front of the camera while the webcam is on. The user then presses one of the defined keys in the events method to save the pictures, which are evaluated in another class. The presence of these images allows the user to control the playing, stopping, and shuffling of the music. When the images are not saved, there is no way to control the music player and no music will play.
+A major part of the program is the events that allow the user to capture images from the webcam. By pressing different keys, the user saves the necessary images in the directory. There are many small parts to the Camera class, including the construction of the surface and the events method that provides the framework for the keys to work. The program moves to another major component of the code, the music player. The most important parts of that code are the methods and functions that play and stop the music when given input. There is also a necessary part that chooses songs randomly from the library where the songs are saved in the directory.
+
+In the webcam part of the project, there were multiple different ways to process the images the user saves. We could have used thresholds, different surface arrays, found blobs in the images, or even taken the average color of a certain area of the screen and returned that value. Ultimately, we chose the existence of certain files in the directory. Originally, we had planned on using hand gestures, but opencv never worked, so we had to move away from that path. It was hard to pick how to process the images because we knew nothing about processing images in these different ways, so we chose the most straightforward path we thought of: using the existence of the images as a condition.
```

Comment:
Computer vision and recognition is tough stuff. Good attempt, and thank you for including the explanation about that here.